README.md

```diff
@@ -76,13 +76,11 @@ In order to run this project you need:
 
 Clone this repository to your desired folder:
 
-### Install
-
-Install any code editor in your computer
+git clone [https://github.com/Bigizi/LeaderBoard]
 
-clone the repo using this command:
+### Install
 
-git clone []
+Install any code editor in your computer the follow the step above to clone the repository
```

Comment:
- [ ] As asked by the previous reviewer please include on your readme file how a non-technical user can clone, install dependencies and run your project, include commands as well